packages/block-editor/src/components/global-styles/color-panel.js

```diff
@@ -228,15 +230,38 @@ function ColorPanelDropdown( {
 							{ 'is-open': isOpen }
 						),
 						'aria-expanded': isOpen,
+						ref: colorGradientDropdownButtonRef,
 					};
 
 					return (
-						<Button __next40pxDefaultSize { ...toggleProps }>
-							<LabeledColorIndicators
-								indicators={ indicators }
-								label={ label }
-							/>
-						</Button>
+						<>
+							<Button { ...toggleProps } __next40pxDefaultSize>
+								<LabeledColorIndicators
+									indicators={ indicators }
+									label={ label }
+								/>
+							</Button>
+							{ hasValue() && (
+								<Button
+									__next40pxDefaultSize
+									label={ __( 'Reset' ) }
+									aria-label={
+										sprintf(
+											/* translators: %s: The name of the color property being reset e.g. "Background" */
+											__( 'Reset %s' ),
+											label.toLowerCase()
+										)
+									}
+									className="block-editor-panel-color-gradient-settings__reset"
+									icon={ resetIcon }
+									onClick={() => {
```

Comment:
Same as above here - spacing seems off. Make sure to run prettier on it (your IDE should ideally do this automatically)

Reply:
Thanks for the heads up. Don't know why the extension wasn't running, and I didn't catch it. It came back after resetting the IDE. It should be fixed now.